src/units.cpp

```diff
@@ -527,9 +562,10 @@ bool Units::equivalent(const UnitsPtr &units1, const UnitsPtr &units2)
     return false;
 }
 
-bool Units::dimensionallyEquivalent(const UnitsPtr &units1, const UnitsPtr &units2)
+bool Units::equivalent(const UnitsPtr &units1, const UnitsPtr &units2)
 {
-    return Units::equivalent(units1, units2) && (Units::scalingFactor(units1, units2) == 1.0);
+    // Units must be compatible to return a non-zero scaling factor
+    return (Units::scalingFactor(units1, units2) == 1.0);
```

Comment:
Remove the surrounding parentheses. Also, full stop at the end of the comment.

Reply:
Done.